### ISSUE TITLE
Перевести проект на copier

### ISSUE THREAD

**Issue by f213** [fandsdev/django#575]
Кажется он гораздо более живой.

https://github.com/copier-org/copier

**Comment by f213** [comment]
Скоро всё будет!

**Comment by nkiryanov** [comment]
@f213 подскажи, а какую проблему решаем, заменой `cookiecutter` на `copier`?
Я бы предложил ничего не делать и оставить cookiecutter.

Доводы за cookiecutter:
1. Он уже работает
2. сложно назвать совсем уж мертвым — последний релиз [чуть больше года назад](https://github.com/cookiecutter/cookiecutter), последний коммит 9 месяцев
3. старше и используется больше. Сомневаюсь, что поддержка пропадёт

А ещё, с учётом трендов, не удивлюсь, что через год вообще потеряет актуальность и будем генерить структуру из Claude Code.

сс @kazqvaizer что думаешь?

**Comment by kazqvaizer** [comment]
Согласен, что cookiecutter вряд ли умрет, но возможно copier просто веселее? Т.е. я бы не сказал, что cookiecutter хорош, мне с ним реально не удобно. Наверное стоит инвестировать немного времени чтобы освоить copier или аналоги и понять, насколько он подходит для нашего шаблона.

**Comment by f213** [comment]
> а какую проблему решаем, заменой `cookiecutter` на `copier`?

Самое главное преиимущество `copier` для меня — более живое развитие. Боюсь, что `cookiecutter` в какой-то момент перестанет работать. Есть ещё пара сомнительных:

- Возможность обновлять уже установленный шаблон, то есть разворачивать поверх рабочего кода. Звучит круто, но на практике всегда будет проще скопировать обновлённый сниппет ручками, чем разбираться с кодом, который он может сломать.
- Заменили JSON на YAML (кек)

В остальном — это та же jinja, только с другими названиями переменных, и с самой главной нерешённой проблемой cookiecutter — необходимостью писать шелл-скрипты для разворачивания. В cookiecutter это [хуки](https://github.com/fandsdev/django/blob/master/hooks/post_gen_project.sh), а в copier их назвали [tasks](https://copier.readthedocs.io/en/stable/configuring/#tasks) и загнали внутрь YAML. До нормального декларативного описания проекта им очень далеко (кажется, как и всем другим скаффолдерам на рынке)

Возможно сейчас в `copier` что-то поменялось конечно, я давно не смотрел. Так что если есть возможность изучить — было бы круто.
